string                              id
string                              agent_name
bool                                assigned
uint64                              deadline
mac_ros_bridge/Item[]               items
